app/views/products/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+ 
+   <div class="container">
+   <%= simple_form_for product do |f| %>
+  <div class="row">
+    <div class="col">
+      <%= f.input :title %>
+    </div>
+    #need to add description 
+  </div>
+  <div class="form-row">
+    <div class="form-group col-md-3">
+      <%= f.input :status %>
+    </div>
+    <div class="form-group col-md-3">
+      <%= f.input :condition %>
```

Comment:
Maybe we can add the collection of the conditions (the ones we decided on yesterday), so the user has a dropdown for this